src/components/form.RadioGroup/form.RadioGroup.tsx

```diff
@@ -0,0 +1,43 @@
+import { Form } from "../form";
+import { useForm } from 'react-hook-form';
+
+type FormData = {
+    diabetesType: string;
+};
+
+
+type DiabetesFormProps = {
+    title: string;
+    description: string;
+    options: string[];
+}
+
+const DiabetesForm = ({title, description, options}: DiabetesFormProps) => {
+    const {register, handleSubmit, formState: { errors }} = useForm<FormData>();
+
+    const onSubmit = (data: FormData) => {
```

Comment:
I like how you validated this, I was curious how this is done on a radio group input 👍🏼

Reply:
Sure, I used the useForm hook from react-hook-form to ensure a radio option is selected (validate the radio group input) before the form can be submitted.